rustls/src/tls13/key_schedule.rs

```diff
@@ -1045,7 +1046,7 @@ test_for_each_provider! {
         let expander = TLS13_AES_128_GCM_SHA256_INTERNAL
             .hkdf_provider
             .expander_for_okm(&traffic_secret);
-        let key = derive_traffic_key(expander.as_ref(), aead_alg.key_len());
+        let key = derive_traffic_key(expander.as_ref(), TLS13_AES_128_GCM_SHA256_INTERNAL.aead_alg);
```

Comment:
Just `aead_alg` here would be equivalent.

Reply:
It seems that `aead::AES_128_GCM` doesn't implement the `Tls13AeadAlgorithm` trait, or do I misunderstand what you mean?

Reply:
Ah, true, it would have to mention `as &dyn Tls13AeadAlgorithm` which is pretty wordy, let's leave it how it is.